### PR TITLE
fft-eval: update to the version with ath11k support

### PR DESCRIPTION
**Pull request by klukonin**
Latest fft-eval has some extra features such as ath11k support, double precision in json signal calculation,   and cleanups (batctl references, minor bugfixes, etc )
This commit is a version and makefile update. Now the native project Makefile do the job (compilation and linking).

Maintainer: Nicolas Pace <nicopace@altermundi.net>
Compile tested: Ubuntu 19.10, current OpenWrt master)
Run tested: Unifi AP AC Mesh, current OpenWrt master, tests done)

Description: Latest fft-eval has some extra features such as ath11k support, double precision in json signal calculation,   and cleanups (batctl references, minor bugfixes, etc )
This commit is a version and makefile update. Now the native project Makefile do the job (compilation and linking).

### CODE REVIEW

File: utils/fft-eval/Makefile [hunk 1]
```diff
@@ -17,23 +17,25 @@ define Package/$(PKG_NAME)
 	CATEGORY:=Utilities
 	MAINTAINER:=Nicolas Pace <nicopace@altermundi.net>
 	URL:=https://github.com/simonwunderlich/FFT_eval
-	TITLE:=Evaluates FFT samples from ath9k driver
+	TITLE:=Evaluates FFT samples from ath9k, ath10k or ath11k driver
 	DEPENDS:= +libc @PACKAGE_ATH_DEBUG @PACKAGE_ATH_SPECTRAL
```
Comment (neheb) [inline utils/fft-eval/Makefile]: That dependency on libc is implicit and should be removed.

**Comment by neheb** [comment]
@aparcar where did this maintainer come from?

**Comment by klukonin** [comment]
Very strange. I didn't close it.
Will do another PR.

**Comment by klukonin** [comment]
This PR was closed by mistake.

New PR is https://github.com/openwrt/packages/pull/11068

**Pull request closed without merging**